docker/patches/increase_ssl_handshake_timeout.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/apps/vmq_server/src/vmq_ranch_config.erl b/apps/vmq_server/src/vmq_ranch_config.erl
```

Comment:
Using GitHub, you'll want to change the actual source code itself, and not add a patch.